docs/en/advanced-concepts/settings.md

```diff
@@ -88,3 +88,5 @@ Note that the actual commands you would use in your test script differ based on
 |`defaultActiveApplication`| Sets the hint for active application selection. This helps WebDriverAgent to select the current application if there are multiple items in the active applications list and the desired one is also one of them. The setting is particularly useful for split-screen apps automation. Defaults to `auto`, which makes WebDriverAgent to select the application whose element is located at `screenPoint` location or a single item from the active apps list if the length of this list is equal to one. | e.g., `com.apple.Preferences` |1.15.0+|
 |`activeAppDetectionPoint`| Defines the coordinates of the current screen point. WebDriverAgent uses this point to detect the active application if multiple application are active on the screen. The format of this value is `x,y`, where x and y are float or integer numbers representing valid screen coordinates. Setting this value to a point outside the actual screen coordinates might corrupt WebDriverAgent functionality. By default the screen point coordinates equal to 20% of the minimum screen dimension each, e.g. `MIN(w, h) * 0.2, MIN(w, h) * 0.2` | e.g. `100,300` |1.15.0+|
 | `includeNonModalElements` | Whether returns all of elements including no modal dialogs on iOS 13+. It fixes [cannot find elements on nested modal presentations](https://github.com/appium/appium/issues/13227), but it might make visibility attributes unreliable. Then, you can also enable `shouldUseTestManagerForVisibilityDetection` settings (defaults to `false`) or `simpleIsVisibleCheck` capability to improve the visibility detection. Defaults to `false`. | `true`, `false` |1.15.0+|
+| `acceptAlertButtonSelector` | Allow to customize accept alert button selector. It helps you to handle an arbitrary element as accept button in `accept alert` command. It supports _class chain_ locator. Defaults to find a button as class type of `XCUIElementTypeAlert`. | <code>**/XCUIElementTypeButton[\`label CONTAINS[c] 'accept'\`]</code> | 1.16.0+ |
```

Comment:
> Defaults to find a button as class type of `XCUIElementTypeAlert`

Not really. I would rather say `The default button location algorithm is used if the provided selector is wrong or does not match any element`